### PR TITLE
Feat(E2E): Festival E2E start Movie Form test.

### PR DESCRIPTION
**Pull request by xmano**
New Movie Form E2E test for Festival App.

### CODE REVIEW

**Comment by xmano** [comment]
Test structure is ready and we can test in the Summary Page:

image

With this I will start the PR review. There are some tests failing and I will fix them along the way.

File: apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts [hunk 1]
```diff
@@ -1,14 +1,174 @@
 /// <reference types="cypress" />
 
-import { clearDataAndPrepareTest } from '@blockframes/e2e/utils/functions';
+import { clearDataAndPrepareTest, setForm, FormOptions, 
+         acceptCookie, uploadFile, createFakeScript, randomID } from '@blockframes/e2e/utils/functions';
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts:4]: lot of imports not needed

File: libs/e2e/src/lib/utils/functions.ts [hunk 2]
```diff
@@ -158,16 +218,11 @@ export function setForm(selector: string, formOpt: FormOptions) {
...
-      if ($formEl.is('mat-select') || ($formEl.attr('role') === 'combobox')) {
-        cy.get('mat-option')
-          .contains(vault).click();
-      }
+      handleFormElement($formEl, keyBag, vault);
     }
     formData.push(formelData);
   })
-  .last().then(() => {
+  .then(() => {
```
Comment (phpgeekfr) [inline libs/e2e/src/lib/utils/functions.ts:225]: maybe more elegant: then(_ => console.table(formData))

Comment (xmano) [reply libs/e2e/src/lib/utils/functions.ts:225]: Sure we can do this. Done

File: apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts (hunk 1 above)
Comment (xmano) [reply apps/festival/festival-e2e/src/integration/dashboard/movie-tunnel.spec.ts:4]: Removed.

**Review by phpgeekfr — APPROVED** [review]

**Pull request merged**